### PR TITLE
Feature/issue 5

### LINKED ISSUE #7: Edit option for task item
From #5 

Story: As a user of the  project, I would like to be able to edit tasks more easily. To achieve this, I would like to have an "edit" button for each task that will show a popup/modal for edit preview. Within this preview, I should be able to update the task name, change the assignee, and change the status. For changing the assignee and status, I would like to be presented with select fields to choose from.

1.Edit button for each task
2. Popup/modal for edit preview.
3. Update task name, change assignee and change the status.
4. Use select field Change assignee and Change status

Note:
Create a branch for this issue -> You can create branch here
image

### PR DESCRIPTION
**Pull request by aktaruzzaman-shaon**

### CODE REVIEW

File: src/Component/TaskItems/TaskItems.js [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
+    // const [object, seObject] = useState({});
+    const [name, setName] = useState('Walking');
+    const [asignee, setAsignee] = useState('Shaon');
+    const [label, setLabel] = useState('Beginner');
+    const [status, setStatus] = useState('Completed');
+    const [modalPreview, setmodalPreview] = useState(false);
+
     const TaskItems = [
         {
-            'name': 'Walking',
```
Comment (rakibdevs) [inline src/Component/TaskItems/TaskItems.js:7]: Why remove this items?

**Review by rakibdevs — CHANGES_REQUESTED** [review]
Create a dynamic task list. Edit each item.

File: src/Component/TaskItems/TaskItems.js (hunk 1 above)
Comment (aktaruzzaman-shaon) [reply src/Component/TaskItems/TaskItems.js:7]: Couldn't work with those data. Need discussion vai.

Comment (rakibdevs) [reply src/Component/TaskItems/TaskItems.js:7]: Let's sit and discuss

**Pull request merged**